components/compliance-service/ingest/events/compliance/helper_methods.go

```diff
@@ -116,13 +117,37 @@ func ReportProfilesFromInSpecProfiles(profiles []*inspec.Profile, profilesSums [
 				}
 			}
 
+			refs := make([]relaxting.ESInSpecReportControlRefs, 0)
+			for _, ref := range control.Refs {
+				var refVal, urlVal string
+				if len(ref.Fields) != 2 {
+					logrus.Warnf("ref object contains more than two fields: %v", ref)
+					continue
+				}
```

Comment:
Does this need to handle the case where it's not 2, where one or both of these values are not present?

Reply:
i think for now it's ok to be strict on this. there's a recommended data format for inspec (ref + uri/url) -- we don't want to reject reports that don't adhere to this b/c there isn't strict typing on the profile format, we just want to support the users that do correctly use the feature